### ISSUE TITLE
Add diagram and adjust Architecture page

### ISSUE THREAD

**Issue by lzdanski** [astronomer/docs#2910]
### Issue Type

Inaccurate, misleading, or out-of-date information

### Links to Affected Docs

- https://docs.astronomer.io/astro/astro-architecture

### The Issue or Context

- We have a new architecture diagram that needs to be added to this page, however, the dark mode version of it is incongruent with the other diagram. We need to make these consistent. 
- After adding the new diagram, we need to adjust the content to better describe/interpret the visual. 

### Fix or Proposed Change

- Add images to the architecture diagram that are both similar visual styles.
- Update the copy to match the architecture diagram.

**Comment by lzdanski** [comment]
@paolaperaza + @manmeetkaur - is this issue a duplicate of this older one? https://github.com/astronomer/docs/issues/281 Can we close the older one?

**Comment by lzdanski** [comment]
Here's some text that can be used as a slack announcement: 

Hi Team, 

We wrapped up a revision of our Architecture page in our docs Overview section this week!

**What changed?**

Previously, the Overview section contained two pages: `Architecture` and `Features`. Now we have replaced `Architecture` with `About Astro`, which covers both the architecture includes new content.

**What’s new?**

We’ve renamed the page title to `About Astro` and added detailed content about the fundamentals components of Astro, including the Cloud UI, the CLI, and Astro Runtime, as well as explanations about the different nuances around how resources are organized on Astro into Deployments, Workspaces, and Organizations. We also added a new diagram that includes the Astro Hypervisor in context with the other elements of Astro.

**Impact**

The page URL has not changed, it is still [`https://docs.astronomer.io/astro/astro-architecture`](https://docs.astronomer.io/astro/astro-architecture), so no redirects or bookmark changes are required. However, the content of the page now describes more than just the architectural components of Astro. If you want to direct users specifically to a high level summary of the Astro Architecture, use [`https://docs.astronomer.io/astro/astro-architecture](https://docs.astronomer.io/astro/astro-architecture)#architecture` instead.
